src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java

```diff
@@ -0,0 +1,176 @@
+
+        String page = Files.lines(Paths.get("dispatcher.html")).collect(Collectors.joining("\n"));
+
+        DispatchData data = new DispatchData();
+
+        boolean upd = InfoReader.updateAll(page, data);
+        int i = 0;
+    }
+
+    public DispatchData getData() {
+        return data;
+    }
+
+    public boolean update(int expiryTime) {
+        try {
+            return update(main.backpage, expiryTime);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+
+    public boolean update(BackpageManager manager, int expiryTime) throws Exception {
+        if (System.currentTimeMillis() <= lastDispatcherUpdate + expiryTime) return false;
+        String page = manager.getConnection("indexInternal.es?action=internalDispatch", Method.GET).getContent();
+
+        if (page == null || page.isEmpty()) return false;
+        lastDispatcherUpdate = System.currentTimeMillis();
+        return InfoReader.updateAll(page, data);
+    }
+
+    public boolean hireRetriever(Retriever retriever) {
+        if (data.getAvailableSlots() > 0) {
+            //retriever.getCost(); // TODO: check cost
+            try {
+                String response = main.backpage.getConnection("ajax/dispatch.php", Method.POST)
+                        .setRawParam("command", "sendDispatch")
+                        .setRawParam("dispatchId", retriever.getId())
+                        .getContent();
+                if (response.contains("\"result\":\"ERROR\"")) {
+                    handleResponse("Failed to dispatch", retriever.getId(), response);
+                } else {
+                    handleResponse("Hired dispatcher", retriever.getId(), response);
+                    return true;
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+                System.out.println("Exception hiring dispatcher: " + e);
+            }
+        }
+        return false;
+    }
+
+    public boolean collect(InProgress progress) {
+        try {
+            if (progress.getCollectable().equals("0")) return false;
+            System.out.println("Collecting: Slot " + progress.getSlotId());
+            String response = main.backpage.getConnection("ajax/dispatch.php", Method.POST)
+                    .setRawParam("command", "collectDispatch")
+                    .setRawParam("slot", progress.getSlotId())
+                    .getContent();
+
+            //parse response
+            //{"result":"OK","message":"Collected the following:","rewardsLog":[{"lootId":"Solidus","amount":3},{"lootId":"PLT-2026","amount":97},{"lootId":"Scrap","amount":3}]}
+
+            if (response.contains("\"result\":\"ERROR\"")) {
+                handleResponse("Unable to collect retriever", progress.getId(), response);
+            } else {
+                handleResponse("Dispatch collected", progress.getSlotId(), response);
+                return true;
+            }
```

Comment:
```suggestion
            return handleResponse("Collected retriever", progress.getId(), response);
```
same as above, just simplify this